### ISSUE TITLE
Feature Request: Copy link address to "primary" too?

### ISSUE THREAD

**Issue by roboshim** [lilydjwg/copy-link-address#1]
Hi @lilydjwg ,
you have done great job! Thank you.

May I ask you to add feature for linux users? On Linux, we have "primary", "secondary" and "clipboard" X selections. This add on stores the text into clipboard, which can be access by many applications using CTRL-V.

But in Linux, we can use "primary" storage, which is available using middle click on mouse. With old firefox with original "A" access key for copy link address, I could use right hand on mouse, right click on link, with left hand press A to copy link and then middle click in xterm or gvim to paste the link. CTRL-V does not work as Paste in xterm and gvim. Even in Firefox or Libreoffice, I can use middle click to paste directly from "primary". Pasting from "clipboard" means either ctrl-v or right click and select paste.

Would it be possible to store link to both, clipboard and primary?

Thank you!

Regads,

Robert.

**Comment by lilydjwg** [comment]
There doesn't seem to exist a webext API for the primary selection, and selection done by scripts doesn't update the primary selection.

**Comment by zevweiss** [comment]
For an alternate workaround that as far as I can tell completely restores Firefox's previous behavior (including setting the primary selection), see Robert Wolfe's comments on bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1701324#c59

(Kudos to @lilydjwg for making this extension though.)

**Comment by zevweiss** [comment]
[...realizing now that @roboshim and Robert Wolf may in fact be the same individual, making this comment a bit silly, haha.]

**Comment by roboshim** [comment]
@zevweiss yes, that's me :smile:  now my roboshim identity is linked to my real name :scream: :rofl:
But it's good to link the omni.ja solution for other linux freaks using primary X selection :slightly_smiling_face: 
And for those, who don't want to play with omni.ja and patch every new FF version, this addon is really great tool! small but very important addon :+1: :slightly_smiling_face:
